docs/openapi/api-internal-pn-bff-pa.yaml

```diff
@@ -0,0 +1,153 @@
+      di notifica inviate e delle notifiche accettate.
+
+paths:
+  '/bff/status':
+    get:
+      summary: Get Status of application
+      operationId: getStatus
+      tags:
+        - StatusTemplate
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/StatusSchema'
+        '500':
+          description: InternalServerError
+          content:
+            application/problem+json:
+              schema:
+                $ref: 'remote-refs.yaml#/components/schemas/Problem'
+
+  '/bff/notifications/sent/{iun}':
+    get:
+      summary: 'Mittente: lettura dettagli notifica versione 2'
+      description: |-
+        Questa operazione permette di leggere tutti i dettagli di una notifica accettata.
+      tags:
+        - SenderReadB2B
+      operationId: getSentNotificationV23                                      # NO EXTERNAL
+      #      operationId: retrieveSentNotificationV23                                # ONLY EXTERNAL
+      parameters:
+        - $ref: '#/components/parameters/uidAuthFleet'            # NO EXTERNAL
+        - $ref: '#/components/parameters/cxTypeAuthFleet'         # NO EXTERNAL
+        - $ref: '#/components/parameters/cxIdAuthFleet'           # NO EXTERNAL
+        - $ref: '#/components/parameters/cxGroupsAuthFleet'       # NO EXTERNAL
+        - $ref: '#/components/parameters/pathIun'
+      responses:
+        '200':
+          description: OK
+          content:
+            "application/json":
+              schema:
+                $ref: '#/components/schemas/BffFullSentNotificationV23'
+        '400':
+          description: Bad request
+          content:
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/Problem'
+        '500':
+          description: Internal error
+          content:
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/Problem'
+
+components:
+
+  parameters:
+    ### PARAMETRI DI AUTENTICAZIONE
+    cxTypeAuthFleet: # NO EXTERNAL
+      $ref: './remote-refs.yaml#/components/parameters/cxTypeAuthFleet'             # NO EXTERNAL
+    cxIdAuthFleet: # NO EXTERNAL
+      $ref: './remote-refs.yaml#/components/parameters/cxIdAuthFleet'               # NO EXTERNAL
+    cxGroupsAuthFleet: # NO EXTERNAL
+      $ref: './remote-refs.yaml#/components/parameters/cxGroupsAuthFleet'           # NO EXTERNAL
+    uidAuthFleet: # NO EXTERNAL
+      $ref: './remote-refs.yaml#/components/parameters/uidAuthFleet'                # NO EXTERNAL
+
+    ### PARAMETRI RECUPERO NOTIFICA
+    pathIun:
+      $ref: './parameters-notification-search.yaml#/components/parameters/pathIun'
+
+  schemas:
+    StatusSchema:
+      $ref: './remote-refs.yaml#/components/schemas/StatusSchema'
+
+    BffFullSentNotificationV23:
```

Comment:
I think no, because FullSentNotification is located into https://github.com/pagopa/pn-delivery/blob/develop/docs/openapi/api-internal-b2b-pa.yaml and it cannot access to `remote-refs.yaml` of pn-delivery. The error:
```
Errors: 
 -Unable to load RELATIVE ref: https:/raw.githubusercontent.com/pagopa/pn-delivery/develop/docs/openapi/remote-refs.yaml
```